2023-04-12 15:15:31,502 ERROR: Exception on /reset_password_request [GET] [in /Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py:1744]
Traceback (most recent call last):
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/wangsj/J/100_Learning/99_playground/Learning-on-The-Flask-Mega-Tutorial/microblog/app/routes.py", line 171, in reset_password_request
    return render_template('reset_password_request.html', title='Reset Password', form=form)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/wangsj/J/100_Learning/99_playground/Learning-on-The-Flask-Mega-Tutorial/microblog/app/templates/reset_password_request.html", line 14, in template
    <p>{{ form.submit() }</p>
jinja2.exceptions.TemplateSyntaxError: unexpected '}'
2023-04-12 15:16:15,024 ERROR: Exception on /reset_password_request [GET] [in /Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py:1744]
Traceback (most recent call last):
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/wangsj/J/100_Learning/99_playground/Learning-on-The-Flask-Mega-Tutorial/microblog/app/routes.py", line 171, in reset_password_request
    return render_template('reset_password_request.html', title='Reset Password', form=form)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/wangsj/J/100_Learning/99_playground/Learning-on-The-Flask-Mega-Tutorial/microblog/app/templates/reset_password_request.html", line 14, in template
    <p>{{ form.submit() }</p>
jinja2.exceptions.TemplateSyntaxError: unexpected '}'
2023-04-12 15:17:12,237 ERROR: Exception on /reset_password_request [GET] [in /Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py:1744]
Traceback (most recent call last):
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py", line 2528, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py", line 1825, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py", line 1823, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/wangsj/J/100_Learning/99_playground/Learning-on-The-Flask-Mega-Tutorial/microblog/app/routes.py", line 171, in reset_password_request
    return render_template('reset_password_request.html', title='Reset Password', form=form)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/flask/templating.py", line 146, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "/Users/wangsj/miniconda3/envs/microblog/lib/python3.9/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/wangsj/J/100_Learning/99_playground/Learning-on-The-Flask-Mega-Tutorial/microblog/app/templates/reset_password_request.html", line 14, in template
    <p>{{ form.submit() }</p>
jinja2.exceptions.TemplateSyntaxError: unexpected '}'
